frontend/src/metabase/components/ButtonBar/ButtonBar.styled.tsx

```diff
@@ -30,6 +30,7 @@ export interface ButtonBarRightProps {
 export const ButtonBarRight = styled.div<ButtonBarRightProps>`
   display: flex;
   flex: ${props => (props.center ? "1 0 0" : "")};
+  gap: 0.5rem;
```

Comment:
(Used in the `ViewFooter`)I removed the margins from each of the individual icons and components and used a gap here instead.